eosclubhouse/system.py

```diff
@@ -1206,6 +1206,15 @@ def get_chassis(klass):
         prop = klass.proxy().get_cached_property('Chassis')
         return prop.unpack()
 
+    @classmethod
+    def get_os(klass):
+        prop = klass.proxy().get_cached_property('OperatingSystemPrettyName')
+        os_name = prop.unpack()
+        os_version = ''
+        if ' ' in os_name:
+            os_name, os_version = os_name.split(' ')
```

Comment:
This will fail to unpack it there are 2 or more space characters in `os_name`. Are you confident that's not the case?
```
>>> foo, bar = 'some random string'.split(' ')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: too many values to unpack (expected 2)
```

Anyways it would be safer to do:
```
if ' ' in os_name:
    return os_name.split(' ')[0], os_name.split(' ')[1]
```

Reply:
I've only checked the name on ubuntu, fedora and endless and there there's no problem, but as you said, this is a bit fragile. I'll try to improve this to make it more robust because I think that this name could be anything and there are a lot of linux distributions.